### PR TITLE
REANA set up

### PR DESCRIPTION
**Pull request by AndriiPovsten**
The set up which is required to run the AGC on the REANA platform.
-  Repr-step(REANA) folder
-  Modified `ttbar_analysis_reana.ipynb`
-  required `file/final_merging.ipynb`
- `Snakefile`
- `reana.yaml file`

### CODE REVIEW

File: analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py [hunk 1]
```diff
@@ -1,46 +1,33 @@
...
-#       format_version: '1.3'
-#       jupytext_version: 1.15.2
-#   kernelspec:
-#     display_name: Python 3 (ipykernel)
-#     language: python
-#     name: python3
-# ---
-
-# %% [markdown]
+#!/usr/bin/env python
```
Comment (alexander-held) [inline analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py:1]: We should do the same jupytext re-sync like in the HEPData case.

File: AGC_clean/analysis-grand-challenge
Comment (alexander-held) [inline AGC_clean/analysis-grand-challenge:1]: This submodule should be removed.

File: analyses/cms-open-data-ttbar/utils/config.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
         # read files from public EOS (thanks to the CMS DPOA team!)
         # note that they are likely only available temporarily
         # and not part of an official CMS Open Data release
-        "INPUT_FROM_EOS": False,
+        "INPUT_FROM_EOS": True,
```
Comment (alexander-held) [inline analyses/cms-open-data-ttbar/utils/config.py:21]: We shouldn't change the default here.

File: analyses/cms-open-data-ttbar/utils/file_input.py [hunk 3]
```diff
@@ -158,4 +158,4 @@ def get_data_rootfiles_uri(self, query, as_signed_url=True, title="Untitled"):
             # update files for each process
             files_per_process.update({process: all_files[parent_key[self.filelist[:,1]==process]]})
 
-        return files_per_process
+        return files_per_process
```
Comment (alexander-held) [inline analyses/cms-open-data-ttbar/utils/file_input.py:161]: Same here as for HEPData: let's not change the newline to minimize files being touched.

File: analyses/cms-open-data-ttbar/Repr-step(REANA)/file_merging.ipynb
Comment (alexander-held) [inline analyses/cms-open-data-ttbar/Repr-step(REANA)/file_merging.ipynb:1]: The merging steps should probably be a script instead of a notebook?

**Comment by AndriiPovsten** [comment]
@alexander-held  the only thing which would require for successful run AGC on REANA is to update the harbor image with e `papermill` package, I'm not sure how it can be done. I could ask Matthew about it.

File: analyses/cms-open-data-ttbar/Repr-step(REANA)/file_merging.ipynb
Comment (AndriiPovsten) [reply analyses/cms-open-data-ttbar/Repr-step(REANA)/file_merging.ipynb:1]: In theory, the workflow can be executed as a script, but the `Snakefile` would be much longer, I believe executing notebooks is sufficient.

**Pull request closed without merging**